playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/rpm_upgrade.yml

```diff
@@ -0,0 +1,6 @@
+- name: Upgrade packages
+  command: "{{ ansible_pkg_mgr}} update -y {{ openshift.common.service_type }}-{{ component }}{{ openshift_version }}"
+
+- name: Ensure python-yaml present for config upgrade
```

Comment:
I haven't completely decided yet.  We could technically add in the oc client download links but I don't view that as a blocker right now all things considered.